### PR TITLE
Introducing Slack notifier

### PR DESCRIPTION
**Pull request by fcamblor**
This notifier is used for new drafts / new submission / new edits / new comments

It allows to publish on 2 different Slack servers :
- 1 dedicated to CFP commitee
- 1 dedicated to Golden Tickets
_This is not something mandatory, if you don't provide a URL for one or both server, then nothing will be sent through Slack_

You need to add a "Incoming Webhooks" custom integration on every concerned server.
By doing this, you will get a webhook URL which will be used in the `SLACKBOT_COMITEE_HOOK_URL` and `SLACKBOT_GT_HOOK_URL`
Incoming_WebHooks___Slack_App_Directory

Details on what is sent can be seen in the [`Slacks`](https://github.com/fcamblor/cfp-devoxx/blob/slack-notifications/app/notifiers/Slacks.scala#L29) class.

Examples :
### New Draft created
Rambox_Pro_-_Perso_-_Home

### Draft Submitted
Rambox_Pro_-_Perso_-_Home

### Submission edited
Rambox_Pro_-_Perso_-_Home

### Private comment posted
Rambox_Pro_-_Perso_-_Home

### Public comment posted (from commitee / speaker)
Rambox_Pro_-_Perso_-_Home
Rambox_Pro_-_Perso_-_Home

### CODE REVIEW

**Comment by fcamblor** [comment]
@nicmarti I hesitated to put this in `ZapActor` but it would have implied to modify some internal of the messaging case classes.

Since I don't know how `ZapActor` is working (does it handle message structure changes over time ?), I didn't want to make this change.

Moreover, I noticed that sometime, I didn't received some emails at some point ... and I wondered if this wasn't due to some delay in processing the actor messages
For instance, when I look at the `sendMessageToCommittee` impl, I can see that `Comment.lastMessageIDForSpeaker()` is used instead of passing comment's id : I was wondering if this was something willed in order to make a sort of "debounce" if multiple messages were posted in a small amount of time.
On the email medium, it would be something understandable ("don't send twice a notification telling that there was 2 messages posted if those 2 messages were posted at the same time") but on the Slack impl, we don't want to "lose" some messages.

**tldr;** if you prefer to move this in an asynchronous place (like `ZapActor`) and this doesn't make any problem, then simply tell me :)

**Comment by fcamblor** [comment]
We need to talk about GT notification policies as well.

If GT would have to be notified (you can easily completely disable this by not providing any slack webhook url for GTs) :
- Should they be notified when a new Draft is created ? (currently implemented : **no**)
- Should they be notified when a new Proposal is submitted ? (currently implemented : **yes**)
- Should they be notified when a Submitted Proposal is edited ? (currently implemented : **yes**) 
- Should they be notified when a new Public message is sent ? (currently implemented : **no**)
- Should they be notified when a new Private message is sent ? (currently implemented : obviously **no**)

Unrelated question : do you think it would be useful to notify comitee on Slack when someone (either from the comitee, or from GT) vote on a talk ?
I didn't implemented it as I don't see a real added value, but it can be easily implemented if needed.

WDYT ?

**Comment by aheritier** [comment]
I don't think we really need to be notified about others votes
I know we have a devoxxfr slack team space but it must be a free one. We could perhaps use it for the committee. Not sure for GoldenTickets

**Comment by fcamblor** [comment]
Agree, on bdxio we created 2 instances per year (1 dedicated to comitee, the other for GTs)

I think it's important if you want to keep your history clean on the organization slack (the bot generates a lot of messages)

**Comment by fcamblor** [comment]
closing as I assume Slack won't be the medium of choice for notifications :-)
